straxen/plugins/peaklets/peaklet_classification_som.py

```diff
@@ -65,67 +65,38 @@ def setup(self):
 
     def compute(self, peaklets):
         # Current classification
-        ptype = np.zeros(len(peaklets), dtype=np.int8)
-
-        # Properties needed for classification:
-        rise_time = -peaklets["area_decile_from_midpoint"][:, 1]
-        n_channels = (peaklets["area_per_channel"] > 0).sum(axis=1)
-        n_top = self.n_top_pmts
-        area_top = peaklets["area_per_channel"][:, :n_top].sum(axis=1)
-        area_total = peaklets["area_per_channel"].sum(axis=1)
-        area_fraction_top = area_top / area_total
-
-        is_large_s1 = peaklets["area"] >= 100
-        is_large_s1 &= rise_time <= self.s1_max_rise_time_post100
-        is_large_s1 &= peaklets["tight_coincidence"] >= self.s1_min_coincidence
-
-        is_small_s1 = peaklets["area"] < 100
-        is_small_s1 &= rise_time < self.upper_rise_time_area_boundary(
-            peaklets["area"],
-            *self.s1_risetime_area_parameters,
-        )
-
-        is_small_s1 &= rise_time < self.upper_rise_time_aft_boundary(
-            area_fraction_top,
-            *self.s1_risetime_aft_parameters,
-            *self.s1_flatten_threshold_aft,
-        )
-
-        is_small_s1 &= peaklets["tight_coincidence"] >= self.s1_min_coincidence
+        peaklets_classifcation = super().compute(peaklets)
 
-        ptype[is_large_s1 | is_small_s1] = 1
-
-        is_s2 = n_channels >= self.s2_min_pmts
-        is_s2[is_large_s1 | is_small_s1] = False
-        ptype[is_s2] = 2
+        peaklet_with_som = np.zeros(len(peaklets_classifcation), dtype=self.dtype)
```

Comment:
In the following are some changes required due to the inheritance and some more explicit naming conventions.